ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_statistical.py

```diff
@@ -317,3 +337,38 @@ def test_bincount(
         weights=x[1],
         minlength=min_length,
     )
+
+
+# percentile
+@handle_test(
+    fn_tree="functional.ivy.experimental.percentile",
+    dtype_and_x=statistical_dtype_values(function="percentile"),
+    keep_dims=st.booleans(),
+    test_gradients=st.just(False),
+)
+def test_percentile(
+    *,
+    dtype_and_x,
+    keep_dims,
+    test_flags,
+    backend_fw,
+    fn_name,
+    on_device,
+    ground_truth_backend,
+):
+    input_dtype, x, axis, interpolation, q = dtype_and_x
+    helpers.test_function(
+        input_dtypes=input_dtype,
+        test_flags=test_flags,
+        ground_truth_backend=ground_truth_backend,
+        fw=backend_fw,
+        fn_name=fn_name,
+        on_device=on_device,
+        a=x[0],
+        q=q,
+        axis=axis,
+        interpolation=interpolation[0],
+        keepdims=keep_dims,
+        rtol_=1e-1,
+        atol_=1e-1,
```

Comment:
Was there a reason you chose to set so high values for the tolerance? A 10% difference in the results would not be acceptable and means there is something wrong with the implementation.

Reply:
I arrived at this value by trial and error of pytests passing, however few of the times I had all the pytests passed at 1e-4, will definitely try to figure this out.